android/gradle.properties

```diff
@@ -18,3 +18,5 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+android.enableAapt2=false
+MYAPP_RELEASE_STORE_FILE=my-release-key.keystore
```

Comment:
@william2958 I think you can remove this form the file. I am also not sure if it should be commited @seland?

Reply:
The enableAapt2 line is a setup line that is specified by the detox get started guide, I am pretty sure this is required to build successfully. The other line however was just me trying to get the CI to pass, it is a meaningless property. It is basically dummy data. I am not sure if this file was to be not committed though